app/components/bread_crumb_nav_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<div>
```

Comment:
Minor nit: can you rename files with `bread_crumb` to `breadcrumb` and classes with `BreadCrumb` to `Breadcrumb`?

Reply:
yes, will do